Gemfile

```diff
@@ -42,7 +42,7 @@ gem 'redis-session-store', github: '18F/redis-session-store', branch: 'master'
 gem 'rqrcode'
 gem 'ruby-progressbar'
 gem 'ruby-saml'
-gem 'saml_idp', git: 'https://github.com/18F/saml_idp.git', tag: 'v0.5.0-18f'
+gem 'saml_idp', git: 'https://github.com/18F/saml_idp.git', branch: 'jmhooper-idp-cert-and-secret'
```

Comment:
We will want to switch this to a new tag once this https://github.com/18F/saml_idp/pull/15 is merged. Until that happens, this is blocked.